beginner/slice.go

```diff
@@ -9,4 +9,11 @@ import (
 func main() {
 	var str string = "Lorem ipsum dolor sit amet"
 	fmt.Println(str[6:11])
+
+	//Make a new slice using make
```

Comment:
Please, add space
```suggestion
	// Make a new slice using make
```
https://golang.org/doc/effective_go.html#commentary

Reply:
Yes. Will add.